app/styles/components/_key-value-table.scss

```diff
@@ -1,7 +1,6 @@
 .key-value-table {
   th, td {
     padding-bottom: 10px;
-    text-transform: capitalize;
```

Comment:
:+1: 

Do u know if I still need to go through other areas of the application and change this behavior?